engine/runtime/examples/fixture/workflow/requirement/plateau4/a001-7/union.yml

```
@@ -17,59 +17,36 @@ graphs:
-        fromPort: default
-        toPort: default
       - id: c81ea200-9aa1-4522-9f72-10e8b9184cb9
         from: 61e89fd2-ea66-4fa1-b426-6f84484a9d40
         to: f5e66920-24c0-4c70-ae16-6be1ed3b906d
         fromPort: area
+        toPort: default
+      - id: c81ea200-9aa1-4522-9f72-10e8b9184cb9
+        from: 61e89fd2-ea66-4fa1-b426-6f84484a9d40
+        to: f5e66920-24c0-4c70-ae16-6be1ed3b906d
+        fromPort: remnants
         toPort: default
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)